src/content/developers/docs/smart-contracts/source-code-verification/index.md

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
Minor nitpick/suggestion - could we shorten this directory name to:
`src/content/developers/docs/smart-contracts/verifying/index.md`?

This would create the page `ethereum.org/en/developers/docs/smart-contracts/verifying/`, keeping it consistent with others e.g.
- https://ethereum.org/en/developers/docs/smart-contracts/testing/
- https://ethereum.org/en/developers/docs/smart-contracts/compiling/
- https://ethereum.org/en/developers/docs/smart-contracts/deploying/

Reply:
I think it's good for consistency as you suggest. As mentioned in the content, verifying usually refers to source code verification, plus we make the distinction with the formal v. at the very beginning. So I'd say 👍

Reply:
Looks like we have a dilemma here. If we didn't have a page on formal verification, then "verifying" would be a good directory name for the page. The best suggestion I can make is using "source-verifying" for this page and "formal verifying" for the page on formal verification.

Also, @kuzdogan, I pushed a new commit to tweak the numbered list outlining instructions for verifying contracts. The initial list didn't put spaces between items, so it showed up as a paragraph in the preview.